src/lib/configureStore.js

```diff
@@ -16,7 +16,7 @@ const middleware = [errorAlert, loginStatusChecker, thunk, soundsMiddleware]
 if (ENV.ENABLE_REDUX_PERF_LOGGING) middleware.push(perfLogger)
 
 const composeEnhancers =
-  typeof window === 'object' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({ name: 'ui' }) : compose
+  typeof window === 'object' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({ name: 'ui', maxAge: 50 }) : compose
```

Comment:
does this increase the number of actions displayed in the redux logger?

Reply:
No, because the default is 50 so it's just making it more obvious which parameter can be modified to change the number of actions displayed. If you increase the number it will show more actions. William is also thinking of removing the core from Redux at some point.